### PR TITLE
update display of metadata frontend to allow for arrays.

### PR DESCRIPTION
**Pull request by jonhartzler**
fixed the front end so it displays arrays

### CODE REVIEW

**Comment by jaronkk** [comment]
I'd like the value functions to be moved to their own components, one for each type

File: app/assets/javascripts/components/metadata/MetadataDate.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+var MetadataDate = React.createClass({
+  displayName: 'Metadata Date',
+
+  propTypes: {
+    metadata_field: React.PropTypes.object.isRequired,
+    label: React.PropTypes.string.isRequired,
+  },
+
+  render: function () {
+    return (<time itemProp={this.props.label} dateTime={this.props.metadata_field.iso8601}>{this.props.metadata_field.value}</time>);
```
Comment (jaronkk) [inline app/assets/javascripts/components/metadata/MetadataDate.jsx]: What is "itemProp" here?

File: app/assets/javascripts/components/metadata/MetadataDate.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+//app/assets/javascripts/components/MetadataDate.jsx
+var React = require('react');
+
+var MetadataDate = React.createClass({
+  displayName: 'Metadata Date',
+
+  propTypes: {
+    metadata_field: React.PropTypes.object.isRequired,
+    label: React.PropTypes.string.isRequired,
```
Comment (jaronkk) [inline app/assets/javascripts/components/metadata/MetadataDate.jsx]: Now that the one itemProp line was removed, do any of these components needs the label prop?

**Comment by jaronkk** [comment]
:+1:

**Comment by jgondron** [comment]
:+1:

**Pull request merged**